mm2src/mm2_db/src/indexed_db/drivers/cursor/cursor.rs

```diff
@@ -317,6 +317,8 @@ impl CursorDriver {
         if matches!(item_action, CursorItemAction::Include) {
             if let Some(cursor_condition) = where_ {
                 if cursor_condition(val.clone())? {
+                    // stop iteration.
+                    self.stopped = true;
```

Comment:
Can we open a PR for this fix alone? this is to merge this bug fix to dev quickly and not wait for reviews or approvals on this PR and this https://github.com/KomodoPlatform/komodo-defi-framework/pull/2066/files